cmp/tab/TabModel.ts

```diff
@@ -94,8 +95,10 @@ export class TabModel extends HoistModel {
     private _renderMode: RenderMode;
     private _refreshMode: RefreshMode;
 
+
     containerModel: TabContainerModel;
     @managed refreshContextModel: RefreshContextModel;
+    @managed errorBoundaryModel: ErrorBoundaryModel;
```

Comment:
good catch -- its leftover from original setup where I was getting this model from context, which I realized was a mistake, as you will end up easily just attaching to the root ErrorBoundaryModel accidentally.  Will remove.

Great cleanups, especially to the docs, and also like the word `clear`.  

I guess one concern I have is how much we want to push using this to display standard http/server exceptions vs. just having it in place for protection against the unexpected react lifecycle exceptions.  With Dash apps especially, we typically already build a specific dash item wrapper to show server exceptions consistently.  I worry that developers and app users will be confused by these two different exception displays....

Reply:
another possibility would be to add an `errorBoundary` prop to panel, (that could be specified as boolean or ErrorBoundaryConfig).  Might make it super easy to insert a boundary in, say, a detail panel.

Which actually makes me wonder whether tab container/navigator/dashcontainer should actually take a similar prop, which would allow apps to customize their built-in boundaries.  

Don't want to over design this, and some of these changes could be added later, but worth thinking about.

Reply:
Its tricky, because the exceptions are so very different -- a server-side/doLoadAsync exception is common, and we still have the ability/desire to render the core UI, previous data, etc.  Rendering the UI allows the users to take specific steps to change parameters etc. to try to recover.

A rendering exception really has to be rendered extremely generically -- we simply can't render our content.  

In any case, the following is a possibility -- and we could even add a method on HoistModel of the form 
`handleExceptionOnParentBoundary()` to make it even easier.

 override async doLoadAsync(loadSpec) {
        try {
            throw XH.exception('my widget data failed to load')
        } catch (e) {
            const errBoundary  = this.lookupModel(ErrorBoundaryModel) as ErrorBoundaryModel;
            errBoundary.handleError(e);
        }
    }